js/tokens.js

```diff
@@ -0,0 +1,496 @@
+        filled = amount - remaining;
+        cost = price * filled;
+        if (market !== undefined) {
+            symbol = market['symbol'];
+            feeCurrency = market['quote'];
+        }
+        fee = {
+            'cost': undefined,
+            'currency': feeCurrency,
+        };
+        let trades = [];
+        for (let i = 0; i < order.trades.length; i++) {
+            trades.push (this.parseTrade (order.trades[i], market));
+        }
+        return {
+            'id': id,
+            'datetime': this.iso8601 (timestamp),
+            'timestamp': timestamp,
+            'lastTradeTimestamp': undefined,
+            'status': status,
+            'symbol': symbol,
+            'type': 'limit',
+            'side': side,
+            'price': price,
+            'cost': cost,
+            'amount': amount,
+            'filled': filled,
+            'remaining': remaining,
+            'trades': trades,
+            'fee': fee,
+            'info': order,
+        };
+    }
+
+    async createOrder (symbol, type, side, amount, price = undefined, params = {}) {
+        await this.loadMarkets ();
+        let market = this.market (symbol);
+        let request = {
+            'tradingPair': market['id'],
+            'amount': this.amountToPrecision (symbol, amount),
+            'side': side,
+            'price': this.priceToPrecision (symbol, price),
+        };
+        let response = await this.privatePostPrivateOrdersAddLimit (this.extend (request, params));
+        // let order = this.parseOrder (response, market);
+        let timestamp = (response['timestamp'] * 1000).toString ();
+        return {
+            'info': response,
+            'datetime': this.iso8601 (timestamp),
+            'timestamp': timestamp,
+            'id': this.safeString (response, 'orderId'),
+        };
+    }
+
+    async cancelOrder (id) {
+        return await this.privatePostPrivateOrdersCancelId ({ 'id': id });
+    }
+
+    async fetchOrderStatus (id) {
+        let order = await this.privateGetPrivateOrdersGetId ({ 'id': id });
+        return this.parseOrderStatus (this.safeString (order, 'orderStatus'));
+    }
+
+    async fetchOrder (id) {
+        await this.loadMarkets ();
+        let order = await this.privateGetPrivateOrdersGetId ({ 'id': id });
+        let parsed = this.parseOrder (order);
+        return parsed;
+    }
+
+    // async fetchMyTrades (symbol = undefined, since = undefined, limit = undefined, params = {}) {}
+
+    // async fetchTransactions (code = undefined, since = undefined, limit = undefined, params = {}) {}
+
+    // async fetchWithdrawals (code = undefined, since = undefined, limit = undefined, params = {}) {}
+
+    // async fetchOpenOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {}
+
+    // async fetchDepositAddress (code, params = {}) {}
+
+    // async withdraw (code, amount, address, tag = undefined, params = {}) {}
+
```

Comment:
Here we have some empty lines, which is also a violation of transpiler rules, can you plz remove all whitespace lines from the body of this method?